project/Dependencies.scala

```diff
@@ -107,6 +109,8 @@ object Dependencies {
 
   val coreSdk = deps ++= coreDeps
 
+  val devtools = deps ++= Seq(typesafeConfig, testContainers)
```

Comment:
Also not ideal, we are adding testcontainers to the classpath. The prod classpath.